packages/react-vis/tests/components/bar-series-tests.js

```diff
@@ -28,14 +28,14 @@ test('BarSeries: Showcase Example - BarChart', t => {
     'should find the right number of bars'
   );
   t.equal(
-    $.find('.vertical-bar-series-example').length,
+    $.find('g.vertical-bar-series-example').length,
```

Comment:
Is the `g` necessary for tests to pass?

Reply:
yes.  The series component is passed the className as well, so the original `find` matched all of the other components as well.

```
console.log($.find('.vertical-bar-series-example').map(x => x.name()))
[ 'VerticalBarSeries', 'BarSeries', 'g' ]
```